flink-libraries/flink-gelly/src/main/java/org/apache/flink/graph/library/HITSAlgorithm.java

```diff
@@ -273,4 +289,23 @@ public void sendMessages(Vertex<K, Tuple2<DoubleValue, DoubleValue>> vertex) {
 			return initVertexValue;
 		}
 	}
+
+	public static class AuthorityEdgeMapper<K, EV> implements MapFunction<Edge<K, EV>, String> {
```

Comment:
Ah when I proposed to label the edges as "authority" or "hub" I didn't really mean to add a `String` label :)
We can do this with a boolean. @greghogan what do you think?